charts/cloudnative-pg/templates/service.yaml

```diff
@@ -26,6 +26,12 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if .Values.service.ipFamilyPolicy }}
+  ipFamilyPolicy: {{ .Values.service.ipFamilyPolicy }}
+  {{- end }}
+  {{- if .Values.service.ipFamilies }}
+  ipFamilies: {{ .Values.service.ipFamilies | toYaml | nindent 2 }}
```

Comment:
Can't test it now, but shouldn't this be on two separate lines?

Reply:
No it works as expected